aad/client.bal

```diff
@@ -16,19 +16,22 @@
 
 import ballerina/http;
 
-# Azure AD Client Object for executing drive operations.
+# This is connecting to the Microsoft Graph RESTful web API that enables you to access Microsoft Cloud service resources.
```

Comment:
The Azure Active Directory connector allows you to connect to the Microsoft Graph RESTful web API. It enables you to access Microsoft Cloud Services resources.